ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot

```diff
@@ -175,6 +175,9 @@ Uninstall Serverless Operator CLI
     END
     ${return_code}    ${output}    Run And Return Rc And Output
     ...    oc delete subscription serverless-operator -n openshift-serverless
+    ${return_code}    ${output}    Run And Return Rc And Output
+    ...    oc delete operatorgroup --all -n serverless-operator --ignore-not-found
```

Comment:
isn't the operatorgroup namespace `openshift-serverless`?

Reply:
i'm referring to the namespace `-n serverless-operator`

Reply:
ah yeah, I just pushed the fix